hs_composite_resource/tests/test_composite_resource.py

```diff
@@ -1352,6 +1352,31 @@ def test_unzip_folder_clash(self):
         # ensure files aren't overwriting name clash
         self.assertEqual(self.composite_resource.files.count(), 3)
 
+    def test_unzip_folder_clash_overwrite(self):
+        """Test that when a zip file gets unzipped a folder with the same
+        name already exists and overwrite is True, the existing folder is overwritten """
+
+        self.create_composite_resource()
+        # add a zip file that contains only one file
+        self.add_file_to_resource(file_to_add=self.zip_file)
+        # resource should have 2 files now
+        self.assertEqual(self.composite_resource.files.count(), 1)
```

Comment:
@sblack-usu comment is wrong which is inconsistent with the corresponding assertEqual() statement

Reply:
the comment is wrong, removed.